services/twilioService.js

```diff
@@ -59,8 +63,9 @@ class TwilioService {
         contact => this.initSingleChannel(contact)
       );
       success = await Promise.all(map);
-    } catch {
+    } catch (er) {
       // TODO: Add sentry logging
+      console.log(`Failed to initiate channel ${er}`);
```

Comment:
maybe we can use `SET_ERROR` https://github.com/Hey-Mentor/mentor-mobile/blob/master/reducers/GeneralReducer.js#L7-L11